code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -80,55 +77,55 @@
 	set name = "Hatch"
 	set category = "Abilities"
 
-	if(world.time < last_special)
-		return
-
-	last_special = world.time + 50 //To prevent button spam.
-
 	var/confirm = alert(usr, "Are you sure you want to hatch right now? This will be very obvious to anyone in view.", "Confirm Regeneration", "Yes", "No")
-	if(confirm == "Yes")
-		if(stat == DEAD) //Uh oh, you died!
-			if(hasnutriment()) //Let's hope you have nutriment in you.... If not
-				if(src) //Runtime prevention.
-					chimera_hatch()
-					visible_message("<span class='danger'><p><font size=4>The lifeless husk of [src] bursts open, revealing a new, intact copy in the pool of viscera.</font></p></span>") //Bloody hell...
-					brainloss += 10 //Reviving from dead means you take a lil' brainloss on top of whatever was healed in the revive.
-					return
-				else
-					return //Runtime prevention
-			else //don't have nutriment to hatch! Or you somehow died in between completing your revive and hitting hatch.
+	if(confirm == "Yes" && reviving)
+
+		//Dead when hatching
+		if(stat == DEAD)
+			//Check again for nutriment (necessary?)
+			if(hasnutriment())
+				chimera_hatch()
+				visible_message("<span class='danger'><p><font size=4>The lifeless husk of [src] bursts open, revealing a new, intact copy in the pool of viscera.</font></p></span>") //Bloody hell...
+				brainloss += 10 //Reviving from dead means you take a lil' brainloss on top of whatever was healed in the revive.
+				return
+
+			//Don't have nutriment to hatch! Or you somehow died in between completing your revive and hitting hatch.
+			else
 				to_chat(src, "Your body was unable to regenerate, what few living cells remain require additional nutrients to complete the process.")
 				reviving = FALSE // so they can try again when they're given a kickstart
-				return
 
-		else if(stat != DEAD) //If they're alive at the time of regenerating.
+		//Alive when hatching
+		else
 			chimera_hatch()
 			visible_message("<span class='danger'><p><font size=4>The dormant husk of [src] bursts open, revealing a new, intact copy in the pool of viscera.</font></p></span>") //Bloody hell...
-			return
-		else
-			return //Runtime prevention.
 
 /mob/living/carbon/human/proc/chimera_hatch()
-	nutrition -= nutrition/2 //Cut their nutrition in half.
-	var/old_nutrition = nutrition //Since the game is being annoying.
+	last_special = world.time + 1 HOUR
```

Comment:
The "reviving" var was what was used for the cooldown, so if you're using the last_special timer instead, remember to set it to false again on hatching.

Reply:
Meant to, somehow removed it.

Reply:
Uh. Just had a check of what last_special is used for elsewhere. This will render the chimera unable to use the resist button for an hour.